build.sbt

```diff
@@ -7,7 +7,9 @@ val compiler = Build.compiler
 val logging = Build.logging
 val loggingJS = Build.loggingJS
 val linker = Build.linker
+val linkerInterface = Build.linkerInterface
```

Comment:
For consistency, these declarations should follow the order of declarations in `project/Build.scala`.